frontend/js/src/explore/ExploreCard.tsx

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import { createRoot } from "react-dom/client";
+import * as React from "react";
+import { useContext } from "react";
+import { get, has } from "lodash";
+import { getPageProps } from "../utils/utils";
+
+import ErrorBoundary from "../utils/ErrorBoundary";
+import GlobalAppContext, { GlobalAppContextT } from "../utils/GlobalAppContext";
+import SimpleModal from "../utils/SimpleModal";
+import APIServiceClass from "../utils/APIService";
+import BrainzPlayer from "../brainzplayer/BrainzPlayer";
+
+type ExploreCardProps = {
+  name: string;
+  desc: string;
+  img_name: string;
+  url: string;
+};
+
+function ExploreCard(props: ExploreCardProps) {
+  const { name, desc, img_name, url } = props;
+  return (
+    <div className="explore-card-container">
+      <div className="explore-card">
+        <a href={`${url}`}>
```

Comment:
We can skip the template string extrapolation here and pass a string directly.
Same goes for line 39 below
```suggestion
        <a href={url}>
```